kinto_client/__init__.py

```diff
@@ -4,6 +4,7 @@
 import uuid
 
 from kinto_client import utils
+from kinto_client.batch import batch_requests
```

Comment:
If you import it here to expose it like ``from kinto_client import batch_request`` you should define ``__all__`` to explicit it.